### PR TITLE
WIP: Adding historical Yugoslavia record(s)

### LINKED ISSUE #45: Don't forget Yugoslavia
https://en.wikipedia.org/wiki/Yugoslavia

We should ensure that we have (n) number of records for the former Yugoslavia if only to kick the tires on the tools around:
- supersedes/superseded_by
- dates – what do we call things?

### PR DESCRIPTION
**Pull request by stepps00**
This PR fixes #45 and includes:

- New `edtf:inception` dates for the six former-Yugoslavian republics (Montenegro, Serbia, Bosnia and Herzegovina, Macedonia, Croatia, and Slovenia)
- Five new Yugoslavia country records (each with a different geometry and dates, based on republic breakaways and declared independence days)
- Linked superseding values in all records.

**To do:**
- [x] Verify all dates are correct and represent the declared independence day of each republic
- [x] Verify that there are no additional changes needed to the `1108955789.geojson` record (breakaway or added areas 1918-1991)
- [x] Connect last feature in "history" to "present" feature
- [x] Verify all properties

### CODE REVIEW

**Comment by stepps00** [comment]
Timeline looks _something_ like...

yugo_flow

File: data/110/895/578/3/1108955783.geojson [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+  "id": 1108955783,
+  "type": "Feature",
+  "properties": {
+    "edtf:cessation":"2006",
+    "edtf:inception":"1992-03~",
```
Comment (nvkelso) [inline data/110/895/578/3/1108955783.geojson]: Your chart in the PR says the cessation date should be: `1992-03-03`

File: data/110/895/578/3/1108955783.geojson [hunk 2]
```diff
@@ -0,0 +1,62 @@
+{
+  "id": 1108955783,
+  "type": "Feature",
+  "properties": {
+    "edtf:cessation":"2006",
```
Comment (nvkelso) [inline data/110/895/578/3/1108955783.geojson]: IRL: the chart is off, refer to blog post instead.

File: data/110/895/578/3/1108955783.geojson (hunk 1 above)
Comment (nvkelso) [inline data/110/895/578/3/1108955783.geojson]: IRL: the chart is off, refer to blog post instead.

File: data/112/611/356/7/1126113567.geojson [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    "lbl:longitude":18.839966,
+    "mps:latitude":44.098986,
+    "mps:longitude":18.839966,
+    "mz:hierarchy_label":1,
+    "mz:is_current":0,
+    "name:eng_x_preferred":[
+        "Socialist Federal Republic of Yugoslavia"
+    ],
+    "name:eng_x_variant":[
+        "Kingdom of Serbs, Croats and Slovenes",
```
Comment (nvkelso) [inline data/112/611/356/7/1126113567.geojson]: I'm dubious of it still being called `"Kingdom of Serbs, Croats and Slovenes",`

File: data/112/611/356/7/1126113567.geojson [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        "Yugoslavia",
+        "Kingdom of Yugoslavia",
```
Comment (nvkelso) [inline data/112/611/356/7/1126113567.geojson]: Ditto

File: data/112/611/356/7/1126113567.geojson [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        "Yugoslavia, Kingdom of",
```
Comment (nvkelso) [inline data/112/611/356/7/1126113567.geojson]: Ditto

File: data/110/895/578/3/1108955783.geojson (hunk 2 above)
Comment (nvkelso) [inline data/110/895/578/3/1108955783.geojson]: Should this be "sometime in 2006" as `2006~`?

Comment (nvkelso) [inline data/110/895/578/3/1108955783.geojson]: Or to match the superseding feature: `2006-05~/2006-06~`?

**Comment by nvkelso** [comment]
Regarding: 

```
1108955783 (Federal Republic of Yugoslavia) superseded by 85633259 (Kosovo), 85633755 (Serbia), 85632667 (montenegro)
```

The other times the country split we've also split the **Federal Republic of Yugoslavia**, so since Kosovo and Serbia didn't really split until 2008, suggest:

- 1108955783 (Federal Republic of Yugoslavia) superseded by **new feature** and 85632667 (montenegro) in 2006
- **new feature** superseded by 85633259 (Kosovo), 85633755 (Serbia) in 2008

File: data/110/895/578/3/1108955783.geojson (hunk 2 above)
Comment (stepps00) [inline data/110/895/578/3/1108955783.geojson]: no work here, chart is off..

File: data/110/895/578/3/1108955783.geojson (hunk 1 above)
Comment (stepps00) [inline data/110/895/578/3/1108955783.geojson]: no work here, chart is off..

File: data/110/895/578/3/1108955783.geojson [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    "geom:latitude":43.865131,
+    "geom:longitude":20.597564,
+    "iso:country":"XX",
+    "lbl:latitude":43.550779,
+    "lbl:longitude":20.94335,
+    "mps:latitude":43.550779,
+    "mps:longitude":20.94335,
+    "mz:hierarchy_label":1,
+    "mz:is_current":0,
+    "name:eng_x_preferred":[
```
Comment (nvkelso) [inline data/110/895/578/3/1108955783.geojson]: Add some alt names:

- State Union of Serbia and Montenegro
- Serbia and Montenegro

**Comment by nvkelso** [comment]
IRL: We skipped adding a new weird interstitial Serbia (plus disputed Kosovo) and instead de-linked Kosovo.

**Comment by nvkelso** [comment]
Over to @thisisaaronland for merging today for tomorrow's blog post about Yugoslavia. 

(Once merged @stepps00 will need to link to the new WOF IDs in the Spelunker, too.)

No PIPing, only property work on existing & current features, and adds `mz:is_current: false` for historic records that don't need PIPing.

**Comment by thisisaaronland** [comment]
This is merged.

**Comment by thisisaaronland** [comment]
Or not... now dealing with a brand new LFS problem... yay!

```
git pull origin staging-work
From github.com:whosonfirst-data/whosonfirst-data
 * branch                      staging-work -> FETCH_HEAD
fatal: meta/wof-timezone-latest.csv: smudge filter lfs failed
```

**Pull request merged**